client/jetpack-cloud/sections/partner-portal/issue-license-v2/licenses-form/hooks/use-product-and-plans.tsx

```diff
@@ -64,6 +64,33 @@ const getProductsAndPlansByFilter = (
 	return allProductsAndPlans || [];
 };
 
+// This function gets the displayable Plans based on how it should be arranged in the listing.
+const getDisplayablePlans = ( filteredProductsAndBundles: APIProductFamilyProduct[] ) => {
+	const plans = getProductsAndPlansByFilter( PRODUCT_FILTER_PLANS, filteredProductsAndBundles );
+
+	const securityPlans = plans.filter( ( { slug } ) => slug.startsWith( 'jetpack-security' ) );
+	const restOfPlans = plans.filter( ( { slug } ) => ! slug.startsWith( 'jetpack-security' ) );
+
+	return [ securityPlans, ...restOfPlans ];
+};
```

Comment:
I think we can make it more generic and scalable rather than restricting it to only filter by `jetpack-security`. WDYT?

```suggestion
const PLAN_FILTERS = [ 'jetpack-security' ];
const getDisplayablePlans = ( filteredProductsAndBundles: APIProductFamilyProduct[] ) => {
	const plans = getProductsAndPlansByFilter( PRODUCT_FILTER_PLANS, filteredProductsAndBundles );

	const filteredPlans = PLAN_FILTERS.map( ( filter ) => {
		return plans.filter( ( { slug } ) => slug.startsWith( filter ) );
	} ).filter( ( subArray ) => subArray.length > 0 ); // Remove empty arrays

	const restOfPlans = plans.filter( ( { slug } ) => {
		return ! PLAN_FILTERS.some( ( filter ) => slug.startsWith( filter ) );
	} );

	return [ ...filteredPlans, ...restOfPlans ];
};
```